apps/backend/src/datasources/postgres/ProposalDataSource.ts

```diff
@@ -409,11 +411,16 @@ export default class PostgresProposalDataSource implements ProposalDataSource {
         }
 
         if (sortField && sortDirection) {
-          if (!fieldMap.hasOwnProperty(sortField)) {
-            throw new GraphQLError(`Bad sort field given: ${sortField}`);
+          if (

```

Comment:
I believe there is a slight problem with this approach. If these fields are excluded then sorting on them will not be done. If you have five 10 or fewer proposals in the system then on the table the sorting will appear to work because sorting happens on frontend, but if you try sorting when you have more proposals only the visible 10 on the first page will be sorted.

I suggest disabling the sorting functionality on the `email` and `last name` in the frontend. Alternatively, we need more elaborate work in the backend to add support for being able to sort these fields

Reply:
Disabling the sorting functionality on email and name in frontend.